src/core/async_graphql_hyper.rs

```diff
@@ -21,7 +22,18 @@ pub trait GraphQLRequestLike {
 #[derive(Debug, Deserialize)]
 pub struct GraphQLBatchRequest(pub async_graphql::BatchRequest);
 impl GraphQLBatchRequest {}
-
+impl Hash for GraphQLBatchRequest {
```

Comment:
Add a todo for the correct implementation for BatchRequest, which should ideally batch execution of individual requests instead of the whole chunk of requests as one.